web/src/lib/components/map/Map.svelte

```diff
@@ -0,0 +1,108 @@
+<script lang="ts">
+	import { MapHelper, createMap } from "./mapUtils";
+	import { onMount } from "svelte";
+	import {} from "ol/ol.css";
+	import type { Layer } from "ol/layer";
+	import LayerItem from "./LayerItem.svelte";
+	import Icon from "../Icon.svelte";
+	import { t } from "../../utils/i8n";
+
+	/**
+	 * Zoom value for map view
+	 * @default 6.5
+	 */
+	export let zoom: number = 6.5;
+
+	/**
+	 * Center latitude of map
+	 * @default 40
+	 */
+	export let lat: number = 40;
+
+	/**
+	 * Center longitude of map
+	 * @default -7
+	 */
+	export let lon: number = -7;
+
+	/**
+	 * Show/Hide layers container
+	 * @default false
+	 */
+	export let layersContainer: boolean = false;
+
+	/**
+	 * MapHelper Object, contains OpenLayers Map object and helper methods
+	 */
+	export let mapHelper: MapHelper;
+
+	const map_id: string = "map_id";
```

Comment:
This should be an exported property of the Map component since who uses this component may want to use a different ID for the target DOM element of where the map will be rendered